cgi/display_org_table.pl

```diff
@@ -73,22 +76,48 @@
 		info: "_TOTAL_ labels",
 		infoFiltered: " - out of _MAX_"
     },
-	paging: false,
+	paging: true,
 	order: [[ 0, "asc" ]],
+	scrollX: true,
+	dom: 'Bfrtip',
+	buttons: [
+        {
+            extend: 'colvis',
+            text: 'Column visibility',
+            columns: ':gt(1)'
+        }
+    ]
 });
 JS
 	;
 
 $request_ref->{scripts} .= <<SCRIPTS
 <script src="https://static.openfoodfacts.org/js/datatables.min.js"></script>
+<script src="https://cdn.datatables.net/buttons/2.3.6/js/dataTables.buttons.min.js"></script>
+<script src="https://cdn.datatables.net/buttons/2.3.6/js/buttons.colVis.min.js"></script>
 SCRIPTS
 	;
 
 $request_ref->{header} .= <<HEADER
 <link rel="stylesheet" href="https://static.openfoodfacts.org/js/datatables.min.css">
+<link rel="stylesheet" href="https://cdn.datatables.net/buttons/2.3.6/css/buttons.dataTables.min.css">
```

Comment:
Could you create a new version of the [datatables.css](https://github.com/openfoodfacts/openfoodfacts-server/blob/1cd785affc00db64a8d27cd7d85e9d726bec5d92/html/js/datatables.css) and so on, which include the buttons? Alternatively, move DataTables to be downloaded from `npm` and bundled with `webpack`.  We should ensure we use a consistent version, and we want to avoid using CDNs our assets.

Reply:
I'm not sure how to go about this @hangy

Reply:
@TheSussex When you look at https://github.com/TheSussex/openfoodfacts-server/blob/9e1dd240ce9a1b4e51e7a3dcd8ac6f03aeaaf57b/html/js/datatables.css#L7-L10, you can see that the datatables we currently publish with some specific settings. If you go to that link, you can select the buttons component to be bundled, too. The simplest version would be to simply download the modified CSS and JS files from the datatables site and then replace the ones currently in git.

Reply:
I did this and it's not showing the column visibility button anymore. Am I supposed to rebuild or what?

Reply:
@hangy

Reply:
> I did this and it's not showing the column visibility button anymore. Am I supposed to rebuild or what?

Yes, can you try rebuilding, please? Just the frontend (`make front_build` if you're using docker) should be enough.

Reply:
This did not work @hangy

Reply:
@hangy @TheSussex I updated the JS files and included the ordering etc. buttons. They were also loaded with a hardcoded path to static.openfoodfacts.org, that's why they were not picked in a local dev environment.

Reply:
Oh nice. Thanks for the update @stephanegigandet